js/player-health-meter.js

```diff
@@ -0,0 +1,46 @@
+//create a health meter as a component
+//will later be changed to be a part of the player object
+let phm = new Component('#health', {
+  data: {
+    points: 100
+  },
+  template: function(props) {
+    return props.points;
+  }
+});
+
+//render to DOM
+// phm.render();
+
+//timer
```

Comment:
Do we really need all this mumbo jumbo commenting? If not, nix it!

Reply:
Removed the comments in player-health-meter.js